sdk/core/core-amqp/tsconfig.src.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../../../tsconfig.lib.json",
   "compilerOptions": {
-    "lib": ["DOM"]
+    "lib": ["DOM", "ES2021.Promise"]
```

Comment:
What is our baseline here?  Are we going ES2021?  ES2022?

Reply:
Good question. I took the liberty to use it because it is [supported since nodejs v15.14](https://node.green/#ES2021-features-Promise-any-AggregateError).